src/ui/component/dropDown/dropDown.tsx

```diff
@@ -0,0 +1,34 @@
+import type { FC } from 'react'
+import { Popover } from '@/ui/component/popover/popover'
+import './dropDown.scss'
+
+export type DropDownProps = {
+  placeholder: string
+  value: string[]
```

Comment:
Not clear to me if the componen might support multiple selections or not...

Reply:
Thanks for pointing this out. It serves as a base for dropdown variants, including dropdown variant with multiselect. We would prefer this component to be more agnostic, instead of coupling it with field `value`. I suggest to remove it for the base dropdown, and provide a props called for example `selections`, for the dropdown multiselect variant.
What do you think?